dao/src/test/scala/za/co/absa/enceladus/dao/rest/JsonSerializerSuite.scala

```diff
@@ -242,6 +242,7 @@ class JsonSerializerSuite extends BaseTestSuite with VersionedModelMatchers {
           |  "dateDisabled": null,
           |  "userDisabled": null,
           |  "parent": null,
+          |  "filter": null,
```

Comment:
Eventually yes, for sure.  But my goal was to limit the filtering to Conformance as much as possible, particularly the mapping tables.
When the filter will be actually possible to set and save, this will need it's enhancement. For now is it acceptable in this limited state?

Reply:
Yes, it makes sense.